test/org/sagebionetworks/bridge/services/AuthenticationServiceTest.java

```diff
@@ -319,12 +320,12 @@ public void invalidDataGroupsAreRejected() throws Exception {
     @Test
     public void secondSignUpTriggersResetPasswordInstead() {
         // Verify that requestResetPassword is called in this case
-        AuthenticationService authServiceSpy = spy(authService);
+        authService = spy(authService);
         
         testUser = helper.getBuilder(AuthenticationServiceTest.class)
                 .withConsent(false).withSignIn(false).build();
-        authServiceSpy.signUp(testUser.getStudy(), testUser.getStudyParticipant());
-        verify(authServiceSpy).requestResetPassword(any(Study.class), any(Email.class));
+        authService.signUp(testUser.getStudy(), testUser.getStudyParticipant());
+        verify(authService).requestResetPassword(any(Study.class), any(Email.class));
     }
```

Comment:
This modifies member variable authService. Past experience tells me that JUnit and TestNG don't reset member variables between tests. This could cause corruption across multiple tests. Revert this test to use a local variable.